src/app/file-manager/reducers/downloadables.reducer.ts

```diff
@@ -11,6 +11,26 @@ export interface DownloadableState {
     items: Item[];
 }
 
+export class DownloadableStateImpl implements DownloadableState {
+    itms: Item[];
+
+    constructor(itm: Item[]) {
```

Comment:
There is no need to use a different spelling here, as members are always prefixed by `this.` so there is no ambiguity. E.g.
```
constructor(items: Item[]) {
  this.items = items;
}
```